staff/johana-ludwig/cv/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Johana Ludwig</title>
+</head>
+<body>
+    <header>
+        <h1>Johana Ludwig</h1>
+        <h2>Online Marketing y Frontend Development</h2>
+        <figure>
+            <img src="" alt="Foto de perfil Johana Ludwig">
+        </figure>
+    </header>
+
+    <main>
+    <section>
+        <table>
+            <tr>
+                <td></td>
+                <td>
+                    <header><h3>Hola, soy Johana Ludwig</h3></header>
```

Comment:
Instead of this <header> here you could use a <th> related to the <td>
check that link: https://www.w3schools.com/tags/att_td_headers.asp